README.md

```diff
@@ -1,2 +1,131 @@
 # analytics-ios-integrations
-Monorepo storing Segment's analytics iOS integrations
+
+Monorepo storing Segment's analytics iOS integrations. You can find additional documentation for this repo in the `guides` directory.
+
+## Requirements:
+To run monorepo or migrate a destination, the following are required:
+
+Xcode 8+
+Xcode command line tools and default must be set under "Locations" in Xcode preferences. (If multiple versions of Xcode are installed on a machine, an occur may occur, and so correct default version must be correctly selected)
+
+And local dependencies:
+
+1- HomeBrew
+2- Cocopods 1.8
+3- RubyGems Bundler
+4- OCLint
+5- Clang-format
+
+Alternatively, run .check.sh to install any missing local dependencies.
+
+```bash
+$ ./check.sh
+```
+
+## Usage
+You can open the iOS-Integrations.xcworkspace project on Xcode. All integrations are included by default.
+
+Any scripts can be either run in "Scripts" directory or in root directory.
+
+To run in root, use the following:
+
+```bash
+$ ./start.sh sh xxxx.sh
+```
+For the purposes of the guide, we will assume, you are inside the Scripts directory.
+
+1- Please run check.sh to make sure all required dependencies have been installed on machine
+```bash
+$ ./check.sh
+```
+
+2- Please run pod install or update in root and in example projects, alternatively you can run to do so for you.
+```bash
+$ ./update.sh
+```
+
+### Linter and formatter
+We use [clang-format](https://github.com/llvm-mirror/clang/tree/master/tools/clang-format) and clang-format's [guide](http://clangformat.com) to format our code and [OCLint](http://docs.oclint.org/en/stable/guide/index.html) to check source code and pod spec lint to check podspec:
+Note: During lint, it includes cleaning project and building source files.
+
+```bash
+$ ./lint  
+$ ./format 
+```
+
+### Tests
+We use xcode build tools to run tests. You can run all the tests using:
+```bash
+$ ./test 
+```
```

Comment:
This still doesn't seem to be working for me: 
```

❌  error: Signing for "Segment-FacebookTests" requires a development team. Select a development team in the Signing & Capabilities editor. (in target 'Segment-FacebookTests' from project 'Segment-Facebook')


Testing failed:
	Signing for "Segment-FacebookTests" requires a development team. Select a development team in the Signing & Capabilities editor.
	Testing cancelled because the build failed.

** TEST FAILED **


❌  error: Signing for "Segment-MixpanelTests" requires a development team. Select a development team in the Signing & Capabilities editor. (in target 'Segment-MixpanelTests' from project 'Segment-Mixpanel')


Testing failed:
	Signing for "Segment-MixpanelTests" requires a development team. Select a development team in the Signing & Capabilities editor.
	Testing cancelled because the build failed.

** TEST FAILED **


❌  error: Signing for "Segment-FirebaseTests" requires a development team. Select a development team in the Signing & Capabilities editor. (in target 'Segment-FirebaseTests' from project 'Segment-Firebase')


Testing failed:
	Signing for "Segment-FirebaseTests" requires a development team. Select a development team in the Signing & Capabilities editor.
	Testing cancelled because the build failed.

** TEST FAILED **
```